docker-compose.yml

```diff
@@ -0,0 +1,46 @@
+version: "3.4"
+
+services:
+    redis:
+        image: redis:latest
+        hostname: redis
+        networks:
+            - main
+        ports:
+            - "6379:6379"
+        restart: always
+    
+    app:
+        build: .
+        env_file: .envrc
```

Comment:
you can't use `.envrc`. the point is that docker compose files only recognize `.env`